controllers/schedule/utils/list.go

```diff
@@ -30,14 +30,15 @@ type ActiveLister struct {
 }
 
 func (lister *ActiveLister) ListActiveJobs(ctx context.Context, schedule *v1alpha1.Schedule) (runtime.Object, error) {
+	const v1alpha1LabelManagedBy = "managed-by"
 	kind, ok := v1alpha1.AllScheduleItemKinds()[string(schedule.Spec.Type)]
 	if !ok {
 		lister.Log.Info("unknown kind", "kind", schedule.Spec.Type)
 		return nil, errors.Errorf("Unknown type: %s", schedule.Spec.Type)
 	}
 
 	list := kind.ChaosList.DeepCopyObject()
-	err := lister.List(ctx, list, client.MatchingLabels{"managed-by": schedule.Name})
+	err := lister.List(ctx, list, client.MatchingLabels{v1alpha1LabelManagedBy: schedule.Name})
```

Comment:
```suggestion
	err := lister.List(ctx, list, client.MatchingLabels{v1alpha1.LabelManagedBy: schedule.Name})
```


you could use `LabelManagedBy` defined in `api/v1alpha1`

Reply:
and remove `const v1alpha1LabelManagedBy = "managed-by"`